subiquity/server/server.py

```diff
@@ -178,6 +178,11 @@ async def free_only_POST(self, enable: bool) -> None:
         # enabling free only mode means disabling components
         self.app.base_model.mirror.disable_components(to_disable, enable)
 
+    async def interactive_sections_GET(self) -> Optional[List[str]]:
+        if self.app.autoinstall_config is None:
+            return None
+        return self.app.autoinstall_config.get('interactive-sections', None)
```

Comment:
I strongly feel like:
```yaml
interactive-sections: ["*"]
```
should be deprecated in favor of
```yaml
interactive-sections: "*"
```
I would like to support the latter going forward while keeping subiquity backward compatible (with a warning emitted if the former construct is used maybe?).

Would it be a good time to initiate this change. - so we don't break the GET /interactive-sections API later on? WDYT

Reply:
I'm fine with that except I want to have the support for both `"*"` and `["*"]` long term, I feel like carrying this syntax for compat reasons is a small burdeon.  Which do you think we should document as the intended syntax?

Reply:
Will explore in future PR